routes/error-tracker.js

```diff
@@ -127,12 +128,18 @@ function handler(req, res) {
     message: normalizedMessage,
     context: {
       httpRequest: {
-        url: req.url.toString(),
+        method: req.method,
+        url: req.originalUrl,
         userAgent: req.get('User-Agent'),
         referrer: referrer,
       },
     },
   };
+
+  if (req.method === 'POST') {
+    event.context.httpRequest.url += '?' + querystring.stringify(params);
```

Comment:
I guess this is the minimal change. Going to test some % diversion first?

Reply:
Probably not necessary, but we can do a diversion in the runtime.

> guess this is the minimal change

Hm?

Reply:
Nevermind.